frontend/src/machines/billings/billingMachine.ts

```diff
@@ -6,6 +6,10 @@ import {
   BillingDetail,
   BillingMember,
 } from "../../utils/fetcher";
+import { QueryClient } from "react-query";
+import { getBillings, getMembers } from "./mockFetcher";
+
+const queryClient = new QueryClient();
```

Comment:
ini queryClient nya jangan bikin baru, perlu import dari App, biar cachingnya ngesync

Reply:
Got it. I'll update it later on